test/SoulboundCreditScore.test.ts

```diff
@@ -87,7 +87,7 @@ const signMintCreditScoreToAddress = async (
     },
     // Types
     {
-      MintCreditScore: [
+      Mint: [
```

Comment:
his will create a breaking change, the SDK uses MintCreditScore for all the credit scores